packages/core/src/NavigationContainer.tsx

```diff
@@ -242,6 +242,16 @@ const Container = React.forwardRef(function NavigationContainer(
   );
 
   React.useEffect(() => {
+    // eslint-disable-next-line no-undef
+    if (__DEV__) {
+      if (
```

Comment:
We'll also need to traverse through the properties in the object and check each property if it's serializable.

Here is an implementation that you can copy: https://github.com/expo/ex-navigation/blob/ec8fa74dc3e94e2386ff6d344892d549d2407185/src/ExNavigationRouter.js#L308